bedrock/mozorg/templates/mozorg/home/home-de.html

```diff
@@ -24,8 +24,8 @@
   {{ css_bundle('protocol-call-out') }}
   {{ css_bundle('home-eu') }}
 
-  {% if switch('homepage-mr2-promo-eu') %}
-    {{ css_bundle('home-mr2-promo') }}
```

Comment:
This also needs to go in an `elif` after the fx100 bundle (and same for the fr template).